client.go

```diff
@@ -8,6 +8,8 @@ const (
 	apiSshkeyBase       = "/objects/sshkeys"
 	apiTemplateBase     = "/objects/templates"
 	apiLoadBalancerBase = "/objects/loadbalancers"
+	apiPaaSBase         = "/objects/paas"
+	apiISOBase          = "/objects/isoimages"
```

Comment:
you're including apiISOBase

Reply:
Oh :( Should I close the request and commit again ? :D